src/pages/Assets/hooks/usePriceHistory/usePriceHistory.test.ts

```diff
@@ -12,7 +12,7 @@ const asset = {
   description: ''
 }
 
-describe('usePriceHistory', () => {
+describe.skip('usePriceHistory', () => {
```

Comment:
on purpose?

Reply:
yee - all the store related tests are going to be completely nuked when we refactor the store anyway.